fiboa_cli/datasets/fr_ec.py

```diff
@@ -0,0 +1,180 @@
+by the Eurocrops project and is available on Zenodo.
+
+EuroCrops is a dataset collection combining all publicly available self-declared crop reporting datasets from countries of the European Union.
+The project is funded by the German Space Agency at DLR on behalf of the Federal Ministry for Economic Affairs and Climate Action (BMWK).
+The work is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License][cc-by-sa].
+
+The project developed a new **Hierarchical Crop and Agriculture Taxonomy (HCAT)** that harmonises all declared crops across the European Union. 
+In the data you'll find this as additional attributes:
+
+| Attribute Name | Explanation                                                 |
+| -------------- | ----------------------------------------------------------- |
+| EC_trans_n     | The original crop name translated into English              |
+| EC_hcat_n      | The machine-readable HCAT name of the crop                  |
+| EC_hcat_c      | The 10-digit HCAT code indicating the hierarchy of the crop |
+"""
+# Bounding box of the data in WGS84 coordinates
+BBOX = [-6.047022416643922, -3.916364769838749, 68.89050422648864, 51.075100624023094] 
+
+# Provider name, can be None if not applicable, must be provided if PROVIDER_URL is provided
+PROVIDER_NAME = "Eurocrops"
+# URL to the homepage of the data or the provider, can be None if not applicable
+PROVIDER_URL = "https://github.com/maja601/EuroCrops/wiki/France"
+# Attribution, can be None if not applicable
+ATTRIBUTION = "Institut National de l'Information Géographique et Forestière"
+
+# License of the data, either
+# 1. a SPDX license identifier (including "dl-de/by-2-0" / "dl-de/zero-2-0"), or
+LICENSE = "CC-BY-2.0"
+# 2. a STAC Link Object with relation type "license"
+# LICENSE = {'title': 'CC-BY-4.0', 'href': 'https://creativecommons.org/licenses/by/4.0/', 'type': 'text/html', 'rel': 'license'}
+
+# Map original column names to fiboa property names
+COLUMNS = {
+    'geometry': 'geometry', #fiboa core field
+    'ID_PARCEL': 'id', #fiboa core field
+    'SURF_PARC': 'area', #fiboa core field
+    'CODE_CULTU': 'code_culture', #fiboa custom field
+    'CODE_GROUP': 'code_group', #fiboa custom field
+    'CULTURE_D2': 'determination_datetime', #fiboa core field - this is a hack to get a fixed date in the field.
+    'EC_trans_n': 'EC_trans_n', #fiboa custom field
+    'EC_hcat_n': 'EC_hcat_n', #fiboa custom field
+    'EC_hcat_c': 'EC_hcat_c' #fiboa custom field
+}
+
+# A list of implemented extension identifiers
+EXTENSIONS = []
+
+# Functions to migrate data in columns to match the fiboa specification.
+# Example: You have a column area_m in square meters and want to convert
+# to hectares as required for the area field in fiboa.
+# Function signature:
+#   func(column: pd.Series) -> pd.Series
+COLUMN_MIGRATIONS = {
+    # replace any value in CULTURE_D2 column with 2018-01-15. This is a complete
```

Comment:
@m-mohr - do you have a better way to deal with this? Basically all I want to do is set every value in the determination_date column as 2018, since I know that's when the data is from, and the 'collection' fails if I don't have a determination_date (for the collection date range).

Happy to make tickets if there's not something I'm just missing, but it seemed like there wasn't a way to just make a new column and set it with a value. I tried the 'migration' ones but couldn't even get that to work.

Also not sure if we should consider allowing the creation of collections without determination_date? I think the thing to do with this one is maybe to just set a collection level determination date (which is an argument for letting any attribute live at collection or feature level, but is the opposite way from what we've mostly been thinking about).

Reply:
...And I just saw a @m-mohr slack comment in #fiboa on cloud native geo slack that I suspect solves this:

Try this where the MIGRATION = NONE is in the template:
```
def migrate(gdf):
    gdf["determination_datetime"] = "2020-01-01T00:00:00Z"
    gdf["determination_method"] = "manual"
    return gdf

MIGRATION = migrate
```

And then you also need to add the new columns to the COLUMNS mapping:
```
COLUMNS = {
   # ... whatever you had here before, and add:
    'determination_method': 'determination_method',
    'determination_datetime': 'determination_datetime'
}
```

Glad I didn't actually file an issue ;) 

But I think it'd be good to mention this in the tutorial, and maybe even have an example in the template.

Reply:
As this came up a couple of times now, I've added a simplification:

```
ADD_COLUMNS = {
  "determination_datetime": "2020-01-01T00:00:00Z",
  "determination_method": "manual"
}
```